urdf/azure_kinect_macro.urdf.xacro

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0"?>
+<!-- 
+ Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License.
+ -->
+
+<robot name="azure_kinect" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <xacro:macro name="azure_kinect" params="prefix:='' parent:='' gazebo:=false *origin">
+
+    <xacro:if value="${parent != ''}">
```

Comment:
Make sure that the indentation is aligned and follows the same style as the old xacro file.